src/config/validation.rs

```diff
@@ -169,15 +172,12 @@ pub fn check_outputs(config: &ConfigBuilder) -> Result<(), Vec<String>> {
             errors.extend(errs.into_iter().map(|msg| format!("Transform {key} {msg}")));
         }
 
-        if transform
-            .inner
-            .outputs(
-                &[(OutputId::dummy(), definition)],
-                config.schema.log_namespace(),
-            )
-            .iter()
-            .map(|output| output.port.as_deref().unwrap_or(""))
-            .any(|name| name == DEFAULT_OUTPUT)
+        if transform_output_ids(
+            transform.inner.as_ref(),
+            "dummy".into(),
```

Comment:
Is this part of a previous TODO we added that involved a lot of changes to the topology code?

Reply:
Bump on this question @StephenWakely

Reply:
It looks like I don't need to use "dummy" here. 

However, this function is only being called to get the names of the ports that the transform will output. The component name is being completely ignored.